python/pyabacus/src/hsolver/py_diago_dav_subspace.hpp

```diff
@@ -130,8 +130,10 @@ class PyDiagoDavSubspace
             std::copy(hpsi_ptr, hpsi_ptr + nvec * ld_psi, hpsi_out);
         };
 
+        hsolver::PreOP<std::complex<double>, base_device::DEVICE_CPU, hsolver::fvec::DivTransMinusEigKernel<std::complex<double>, base_device::DEVICE_CPU>>
```

Comment:
The name "DivTransMinusEigKernel" is a disaster.

Reply:
Sorry but do you have any better idea?

Reply:
The function should be named with following considerations: precisely describe what it does. In present case, you mix the mathematical operation with some rare numerical trick like "kernel". So there will be quite few people can know what this function does from its name. You should either name the function purely with names mathematical operations, or use human natural language describe the action that what quantity is calculated, additionally from what quantity.